remoteworkitem/trackerquery_repository_blackbox_test.go

```diff
@@ -30,10 +31,7 @@ func (s *trackerQueryRepoBlackBoxTest) SetupTest() {
 
 func (s *trackerQueryRepoBlackBoxTest) TestFailDeleteZeroID() {
 	// Create at least 1 item to avoid RowsEffectedCheck
-	tr, err := s.trRepo.Create(
-		s.Ctx,
-		"http://api.github.com",
-		remoteworkitem.ProviderGithub)
+	testFxt := tf.NewTestFixture(s.T(), s.DB, tf.Trackers(1))
```

Comment:
Not very important but please use `fxt` instead `testFxt`. We use `fxt` almost everywhere else.

Reply:
@kwk done here -> 7c011dd